packages/components/psammead-paragraph/src/index.stories.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
```suggestion
import { storiesOf } from '@storybook/react';
```
`no-extraneous-dependencies` isn't needed in psammead due to https://github.com/BBC-News/psammead/blob/latest/.eslintrc.js#L25